README.md

```diff
@@ -4,12 +4,47 @@
 
 Web client for [`rezervo`](https://github.com/mathiazom/rezervo), including booking schedules and user preferences.
 
-### Setup
+### 🧑‍💻 Development
 
-Define your own `.env.local` from `.env.local.example`
+#### 🧑‍🔧 Setup
 
+1. Install dependencies using Yarn
+   ```shell
+   yarn install
+   ```
+2. Define your own `.env.local` from `.env.local.example`
+   ```shell
+   # If you want [on-demand revalidation](https://nextjs.org/docs/basic-features/data-fetching/incremental-static-regeneration#on-demand-revalidation), make sure to define `REVALIDATION_SECRET_TOKEN` in `.env.local`
+   cp .env.local.example .env.local
+   ```
```

Comment:
Comment should be moved out of code block and included as a Markdown comment (`>`)

Reply:
Good point, fixed